nbs/examples/MLFlow_and_neuralforecast.ipynb

```diff
@@ -0,0 +1,429 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    plot_df = pd.concat([Y_df, Y_hat_df]).set_index('ds')
</pre><p>I think this should be Y_train_df and Y_hat_df. But can you please instead use something like this:</p>

<pre class="ql-syntax" spellcheck="false">from utilsforecast.plotting import plot_series

plot_series(Y_train_df, Y_hat_df)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Changed Y_df with Y_df_train via commit <a href="https://github.com/Nixtla/neuralforecast/pull/892/commits/eff6207acd593c67cc325f1bdb913fdfcebcd259" target="_blank">https://github.com/Nixtla/neuralforecast/pull/892/commits/eff6207acd593c67cc325f1bdb913fdfcebcd259</a></p><p> </p><p> </p><p>Note: I tried using the utilsforecast library as suggested but when I used above code I get the following error:</p><p> </p><p>KeyError: 'unique_id'</p><p> </p><p> </p><p>If I just change the dataframe, it works.</p>

Reply:
<p>You don't have to concatenate the dataframes, as I suggested just <code>plot_series(Y_train_df, Y_hat_df)</code> should work without any issues (both should have the id as a column but I believe that's the case with your current code).</p>

Reply:
> You don't have to concatenate the dataframes, as I suggested just `plot_series(Y_train_df, Y_hat_df)` should work without any issues (both should have the id as a column but I believe that's the case with your current code).

Hello @jmoralez ,

I tried this sugesstion exactly like this and it didn't work (I got the KeyError: 'unique_id'), so I tried changing the unique id number on Y_train_df, that didn't work, then when I concatenated, it did plot but it is not plotting the NBEATS prediction.

Shall we release this a it is written right now and then add this suggestion, including the utilsforecast plot_series method, with our next update for this notebook? 

Cheers!